classes/phing/listener/DefaultLogger.php

```diff
@@ -159,20 +159,22 @@ public function buildStarted(BuildEvent $event)
      */
     public function buildFinished(BuildEvent $event)
     {
+        $msg = PHP_EOL . $this->getBuildSuccessfulMessage() . PHP_EOL;
         $error = $event->getException();
-        if ($error === null) {
-            $msg = PHP_EOL . $this->getBuildSuccessfulMessage() . PHP_EOL;
-        } else {
+
+        if ($error !== null) {
             $msg = PHP_EOL . $this->getBuildFailedMessage() . PHP_EOL;
+
             self::throwableMessage($msg, $error, Project::MSG_VERBOSE <= $this->msgOutputLevel);
         }
+
         $msg .= PHP_EOL . "Total time: " . self::formatTime(Phing::currentTimeMillis() - $this->startTime) . PHP_EOL;
 
         if ($error === null) {
             $this->printMessage($msg, $this->out, Project::MSG_VERBOSE);
-        } else {
-            $this->printMessage($msg, $this->err, Project::MSG_ERR);
         }
+
+        $this->printMessage($msg, $this->err, Project::MSG_ERR);
```

Comment:
This will **always** print an error message!

Reply:
OMG! Really! I will put a ternary.

Reply:
Would you mind adding a test for this method, please.